Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[workspace]
+members = ["db", "cli"]
+resolver = "2"
+
+[workspace.dependencies]
+color-eyre = "0.6.3"
+sqlx = { version = "0.7", default-features = false }
+tokio = { version = "1", features = ["full"] }
+tracing = "0.1.40"
+url = "2.5.0"
+uuid = { version = "1.8.0", features = ["v4"] }
+
+[workspace.lints.clippy]
+pedantic = "deny"
+missing_errors_doc = "allow"
```

Comment:
We are turning on `pedantic` mode in Clippy which means it turns on lots more extra lints! If these get annoying we can turn some off, but good to start with all the best practices!